### PR TITLE
fix: lets default to recreate the vault bucket

### PR DESCRIPTION
**Pull request by jstrachan**
which makes installing vault more reliable

### CODE REVIEW

File: pkg/cloud/gke/vault/vault_backend.go [hunk 1]
```diff
@@ -80,21 +84,35 @@ func CreateGCPServiceAccount(kubeClient kubernetes.Interface, vaultName, namespa
...
+	exists, err := gke.BucketExists(projectID, bucketName)
 	if err != nil {
 		return "", errors.Wrap(err, "checking if Vault GCS bucket exists")
 	}
 	if exists {
-		return bucketName, nil
+		if !recreate {
+			return bucketName, nil
+		}
+		if batchMode {
```
Comment (ccojocar) [inline pkg/cloud/gke/vault/vault_backend.go]: I think, the condition should be `!bachMode` when prompting.

Comment (jstrachan) [reply pkg/cloud/gke/vault/vault_backend.go]: thanks - doh - fixed!

**Comment by jstrachan** [comment]
@ccojocar fixed that boolean issue you spotted btw ;)

**Pull request merged**